unpackaged/config/trial/package.xml

```diff
@@ -131,7 +131,6 @@
         <members>LeadSource</members>
         <members>OpportunityStage</members>
         <members>OpportunityType</members>
-        <members>Salutation</members>
```

Comment:
I think there is still an issue with the trial/package.xml and standardValueSet. The PR shouldn't have any reference to these two files at all. If they're included here it's because the PR is trying to modify them which it shouldn't be doing at all. The files should be left as they are.